app-backend/mongo/data/resolvers/user.js

```diff
@@ -0,0 +1,13 @@
+import * as dbUsers from '../mongo/user';
+
+export const userResolver = {
+    Query: {
+        users: () => dbUsers.getUsers(),
+        getUserById: (_, args) => dbUsers.getUserById(args._id),
```

Comment:
Curious, what is the reason is for creating `getUserById`, `getUserAvailabilityById` and `getUserByUsername`, why not pass args to `users`, and in your users resolvers, pass args to Mongoose?